drivers/include/sx127x.h

```diff
@@ -95,6 +95,12 @@ extern "C" {
 #ifndef SX127X_DIO_PULL_MODE
 #define SX127X_DIO_PULL_MODE             (GPIO_IN_PD) /**< pull down DIOx */
 #endif
+
+/** @brief Frequency step to calculate carrier frequency from internal values */
+#ifndef SX127X_FREQUENCY_STEP
+#define SX127X_FREQUENCY_STEP      (61.03515625)
```

Comment:
31250 / 512 ?

Reply:
The value comes from (32 * 10^6)/(2^19)

Reply:
Or 15625/256

Reply:
sure, I can change it

Reply:
Maybe even
```C
#define SX127X_FREQUENCY(chan) ((chan * 15625) >> 8)
```

Reply:
the last one only works for converting the internal value (`chan` variable) to the actual carrier frequency. Usually we need the other way around (we know the carrier frequency and we want to get the actual internal value()